src/myrtlespeech/builders/fully_connected.py

```diff
@@ -54,10 +54,15 @@ def build(
     if fully_connected_cfg.hidden_size > 0:
         hidden_size = fully_connected_cfg.hidden_size
 
+    dropout = 0
+    if fully_connected_cfg.dropout > 1e-8:
+        dropout = fully_connected_cfg.dropout
```

Comment:
This was to avoid the proto float value which gives non-zero value when not set due to numerical error. Have updated the proto to use `FloatValue` to avoid this